api/orchestrator/orchestrator.proto

```diff
@@ -39,6 +39,14 @@ option go_package = "clouditor.io/clouditor/api/orchestrator";
 
 // Manages the orchestration of components within the Clouditor architecture
 service Orchestrator {
+  // Get Runtime
+  rpc Runtime(RuntimeRequest)
```

Comment:
GetRuntime would be a fitting request I would say. The response could then be actually named `Runtime` without any wrapping (since we allow this for `Get` calls)